atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -143,11 +143,9 @@ def set_help(self, extra, worker_metadatas):
                 extra['image']['help'] = None
 
     def set_media_types(self, extra, worker_metadatas):
```

Comment:
Right, reworked this - first `pulp_sync` is checked and then `worker_annotations`